css/style.css

```diff
@@ -9,400 +14,633 @@ body {
+  position: absolute;
+  width: 400px;
+  height: 41px;
+  left: 55px;
+  top: 40px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 41px;
+  color: #000000;
+ }
+
+ .bag__title {
+  position: absolute;
+  width: 220px;
+  height: 17px;
+  left: 55px;
+  top: 98px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 16px;
+  letter-spacing: 0.07em;
+  color: #000000;
+  opacity: 0.5;
+}
+
+.bag-text {
+  position: absolute;
+  width: 368px;
+  height: 148px;
+  left: 55px;
+  top: 179px;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 21px;
+  color: #000000;
+  opacity: 0.5;
+}
+
+.bag__color {
+  position: absolute;
+  top: 375px;
+  left: 55px;
+}
+
+.color-wight,
+.color-blue,
+.color-green {
+  width: 30.18px;
+  height: 30px;
+}
+
+.color-pink {
+  width: 38px;
+  height: 38px;
+}
+
+.color-wight {
+  background:#DEDEDE;
+}
+
+span {
+  position: absolute;
+  width: 125px;
+  height: 57px;
+  left: 288px;
+  top: 365px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 56px;
+  color: #000000;
+}
+
+.button-minus,
+.button-plus {
+  border: none;
+}
+
+.button-number {
+  position: absolute;
+  width: 100px;
+  height: 44px;
+  left: 55px;
+  top: 468px;
+  border: 1.5px solid #000000;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+}
+
+.bag__button {
+  position: absolute;
+  width: 253px;
+  height: 44px;
+  border: 1.5px solid #000000;
+  box-sizing: border-box;
+  left: 170px;
+  top: 468px;
+  background: #FFFFFF;
+}
+
+.bag__link {
+  position: absolute;
+  width: 299px;
+  height: 58px;
+  margin-left: 884px;
+  top: 1663px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+  line-height: 16px;
+  color: #000000;
+}
+
+.bag__arrow {
+ position: absolute;
+ width: 241px;
+ left: 1407px;
+ top: 1665px;
 }
 
-@media (max-width: 1100px) { 
-    .new-сollection__main-button {
-        width: 35%;
+/* Медиа запросы */
+
+@media (max-width: 3000px) { 
+  .new-сollection__social {
+      width: 50%;
     }
-}
+ }   
 
-@media (max-width: 1100px) { 
-    .new-сollection__main-link {
-        width: 50%;
-        margin-left: 3%;
+@media (max-width: 1500px) { 
+  .new-сollection__social {
+      width: 45%;
     }
-}
+} 
 
-@media (max-width: 1100px) { 
-     .new-сollection-arrow {
-        width: 10%;
-    } 
+@media (max-width: 1500px) { 
+  .title {
+      top: 30%;
+      width: 20%;
+      font-size: 50px;
+      line-height: 52px;
+      left: 55%;
+    }
+} 
+ 
+ @media (max-width: 1500px) { 
+   .text {
+       width: 35%;
+       left: 55%;
+       height: 60%;
+     }
+ }
+ 
+ @media (max-width: 1500px) { 
+   .new-сollection__main-card {
+       width: 38%;
+       left: 55%;
+       margin-top: 1%;
+     }
+ }
+     
+ @media (max-width: 1500px) { 
+   .girl {
+       width: 50%;
+       right: 60%;
      }
+ }
 
+ @media (max-width: 1500px) { 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline after "{" of a multi-line block (block-opening-brace-newline-after)](https://app.codacy.com/gh/WebHeroSchool/Love-metal_LOCO/pullRequest?prid=5478856)